planner/cascades/testdata/integration_suite_in.json

```diff
@@ -148,7 +148,9 @@
     "name": "TestInlineProjection",
     "cases": [
       "select /*+ HASH_JOIN(t1) */ t1.b, t2.b from t1, t2 where t1.a = t2.a;", // TODO: need column pruning again.
+      "select /*+ HASH_JOIN(t1) */ t1.b, t2.b from t1, t2 where t1.a = t2.a and t1.a = 1",
       "select /*+ HASH_JOIN(t1) */ t1.b, t2.b from t1 inner join t2 on t1.a = t2.a;",
+//      "select /*+ HASH_JOIN(t1) */ t1.b, t2.b from t1 inner join t2 on t1.a = t2.b and t1.a = 1",
```

Comment:
this is for the issue [#20460 ](https://github.com/pingcap/tidb/issues/20460). After that fixed, this should work.